rocAL/rocAL_pybind/example/new_api/rocAL_api_video_pipeline.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import torch
+from amd.rocal.pipeline import Pipeline
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+import numpy as np
+from parse_config import parse_args
+
+class RALIVideoIterator(object):
+    """
+    RALIVideoIterator for pyTorch.
+
+    Parameters
+    ----------
+    pipelines : list of amd.rali.pipeline.Pipeline
+                List of pipelines to use
+    size : int
+           Epoch size.
+    """
+
+    def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False, multiplier=None, offset=None, tensor_dtype=types.FLOAT, display=False ,sequence_length=3):
+
+        try:
+            assert pipelines is not None, "Number of provided pipelines has to be at least 1"
+        except Exception as ex:
+            print(ex)
+
+        self.loader = pipelines
+        self.tensor_format = tensor_layout
+        self.multiplier = multiplier if multiplier else [1.0, 1.0, 1.0]
+        self.offset = offset if offset else [0.0, 0.0, 0.0]
+        self.reverse_channels = reverse_channels
+        self.tensor_dtype = tensor_dtype
+        self.batch_size = self.loader._batch_size
+        self.w = self.loader.getOutputWidth()
```

Comment:
these are image specific parameters. Can you specify video sequence length and batch_size etc.. The sample has to be clear that it is iterating video files